src/components/MetricsPane/charts/AggregateHardCoralCover.jsx

```diff
@@ -57,29 +60,34 @@ export const AggregateHardCoralCover = () => {
     })
     .filter((record) => record !== undefined)
 
-  const markerColors = hardCoralAveragesPerSurvey.map((_, index) => {
-    const colors = chartTheme.aggregateCharts.hardCoralCover.marker
+  const bins = Array.from({ length: (end - start) / binSize }, (_, i) => start + i * binSize)
 
-    if (index <= 5) {
-      return colors.low
+  const binColors = bins.map((bin) => {
+    const markerColors = chartTheme.aggregateCharts.hardCoralCover.marker
+
+    if (bin < 10) {
+      return markerColors.low
     }
-    if (index > 5 && index <= 15) {
-      return colors.medium
+
+    if (bin >= 10 && bin < 30) {
+      return markerColors.medium
     }
 
-    return colors.high
+    return markerColors.high
   })
 
-  const plotlyDataConfiguration = [
-    {
-      x: hardCoralAveragesPerSurvey,
-      xbins: { start: 0, end: 100, size: 2 },
+  const plotlyDataConfiguration = bins
+    .map((bin, index) => ({
+      x: hardCoralAveragesPerSurvey.filter((val) => val >= bin && val < bin + binSize),
```

Comment:
Im thinking this conditional would be easier to immediately understand with the above `binStartValue` naming (or something similarly specific if I am incorrect about it being a start value :) )